models/species.schema.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const speciesSchema = new mongoose.Schema({
+  _id: String,
+  name: { type: String, required: true, unique: true },
+  code: { type: String, required: true, unique: true },
+  isEndangered: { type: Boolean, default: false },
+  isAssigned: { type: Boolean, default: false },
```

Comment:
What is this for? don't think we need it

Reply:
Hi Chris, could you specify which line(s) you are referring to that we don't need?

Reply:
I assume you're referencing line 8 or are you referencing all the lines? the isAssigned is supposed to correlate to the additional species bubble from the add new species pop up in the figma